oarepo_ui/theme/assets/semantic-ui/less/oarepo_ui/themes/default/modules/field_label.variables

```diff
@@ -0,0 +1,30 @@
+/*******************************
+            Theme
+*******************************/
+
+/*
+  The type is the type of the component as defined in semantic-ui - element, collection, view, module
+  For simple component, 'element' is the correct type
+*/
+@type: "modules";
+
+/*
+  This is the name of the component - though not required, keep it the same as the filename
+*/
+@element: "cf_field_label";
```

Comment:
just field_label

Reply:
@mirekys My bad I put wrong stuff in .variables....